src/Facets/GnosisBridgeFacet.sol

```diff
@@ -2,42 +2,47 @@
 pragma solidity ^0.8.17;
 
 import { ILiFi } from "../Interfaces/ILiFi.sol";
-import { IXDaiBridge } from "../Interfaces/IXDaiBridge.sol";
+import { IGnosisBridgeRouter } from "../Interfaces/IGnosisBridgeRouter.sol";
 import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
 import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
-import { InvalidSendingToken, NoSwapDataProvided } from "../Errors/GenericErrors.sol";
 import { SwapperV2, LibSwap } from "../Helpers/SwapperV2.sol";
 import { Validatable } from "../Helpers/Validatable.sol";
+import { InvalidConfig, InvalidSendingToken } from "../Errors/GenericErrors.sol";
 
 /// @title Gnosis Bridge Facet
 /// @author LI.FI (https://li.fi)
-/// @notice Provides functionality for bridging through XDaiBridge
-/// @custom:version 1.0.0
+/// @notice Provides functionality for bridging through Gnosis bridge router
+/// @custom:version 2.0.0
 contract GnosisBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
     /// Storage ///
 
     /// @notice The DAI address on the source chain.
```

Comment:
nitpick: instead of source chain we could say mainnet as this facet is only deployed there and the statement is only true for mainnet

Reply:
thats true, updated